packages/shared/components/Drawer.svelte

```diff
@@ -55,9 +56,21 @@
         let x: number
         let y: number
         let init: number
+        let startY: number
+        let now: number
+        // Define arrays for calc velocity later
+        const positionQueue = [0, 0, startY]
+        const timeQueue = [0, 0, now]
```

Comment:
startY and now are unused and initialising the arrays below with [0, 0, undefined], maybe not necessary?

Reply:
Good catch! yes agree it can be removed, I'll check it, thanks!